applications/se/pom.xml

```diff
@@ -53,6 +53,11 @@
                                     <goal>compile</goal>
                                 </goals>
                                 <phase>package</phase>
+                                <!-- configuration>
+                                    <buildArgs>
```

Comment:
Why is this commented-out ?

Reply:
I guess we could keep that in for SE, will uncomment.

Reply:
cannot uncomment, there is a bug in native image right now that causes this to fail. Removing from code (for now)